alacritty_terminal/src/renderer/rects.rs

```diff
@@ -81,11 +107,11 @@ impl RenderLines {
         Self::default()
     }
 
-    pub fn into_rects(self, metrics: &Metrics, size: &SizeInfo) -> Vec<RenderRect> {
+    pub fn as_rects(&self, metrics: &Metrics, size: &SizeInfo) -> Vec<RenderRect> {
```

Comment:
Same here. I don't like when `as_` is a linear thing.

Reply:
I've decided to go for just `rects`, since it doesn't really convert anything. The return signature isn't `-> RenderRect`, so it's not a direct conversion between types anyways.